iceoryx2-bb/elementary/src/lib.rs

```diff
@@ -64,3 +62,12 @@ pub enum CallbackProgression {
     /// Continues the iteration
     Continue,
 }
+
+// Re-exports
+
+pub mod placement_default {
```

Comment:
I would prefer to remove those re-exports. I trust the CI to catch every issue in this case.

`PlacementDefault` is re-exported in `iceorxy2::prelude::` so it would not change the public end-user API.

Can you please ensure that `ZeroCopySend` is re-exported in prelude as well. I have  a feeling that I have overlooked this in previous PR reviews.

Reply:
`ZeroCopySend` is re-exported. I added the re-exports to minimize the changes in the PR. I can remove then later. Currently it ensures that everything looks the same for the other crates.